db/1.13/modules/ROOT/pages/database-connector-udt-stored-procedure.adoc

```diff
@@ -203,14 +242,14 @@ Db::prepareArray("PEOPLE", payload map (item, index) -> ( toPerson(item) ) )
 .Transform Message configuration
 image::database-oracleudt-example-6.png[Transform Message configuration]
 
-The `prepareStruct` instructs Mule to create JDBC struct objects based on the `Name` type and their corresponding properties when a database operation is performed. In the example, types `PERSON` and `PHONE_NUMBER` are struct objects. The function expects the following parameters in order:
+When the Mule app performs a Database Connector operation, the `prepareStruct` instructs Mule to create JDBC struct objects based on the `Name` type and their corresponding properties. In the example, types `PERSON` and `PHONE_NUMBER` are struct objects. The function expects the following parameters in order:
```

Comment:
This doesn't happen when the Mule app performs an operation, is a DW function used to "prepare" data that will be sent to the db at a later time. The original text tried to account for this difference.